src/components/KyberAITokenBanner/index.tsx

```diff
@@ -0,0 +1,252 @@
+    { address: token1?.address, chain },
+    { skip: !token1?.address || !account || !isWhiteList, refetchOnMountOrArgChange: true },
+  )
+
+  const { data: tokenNativeOverview, isFetching: fetching2 } = useTokenDetailQuery(
+    { address: NativeCurrencies[chainId].wrapped.address, chain },
+    { skip: !account || !isWhiteList, refetchOnMountOrArgChange: true },
+  )
+
+  const isFetching = fetching0 || fetching1 || fetching2
+
+  const token: ITokenOverview | undefined = useMemo(() => {
+    if (isFetching) return undefined
+
+    const token =
+      tokenInputOverview?.kyberScore && tokenInputOverview.kyberScore.label !== ''
+        ? tokenInputOverview
+        : tokenOutputOverview?.kyberScore && tokenOutputOverview.kyberScore.label !== ''
+        ? tokenOutputOverview
+        : tokenNativeOverview?.kyberScore && tokenNativeOverview.kyberScore.label !== ''
+        ? tokenNativeOverview
+        : undefined
+
+    return token
+  }, [isFetching, tokenInputOverview, tokenOutputOverview, tokenNativeOverview])
+
+  if (!token || !isWhiteList || !account) return null
+
+  const color = calculateValueToColor(token?.kyberScore?.score || 0, theme)
+  return (
+    <Wrapper>
+      {above768 ? (
+        <Container
+          color={color}
+          onClick={() => {
+            mixpanelHandler(MIXPANEL_TYPE.KYBERAI_SWAP_INSIGHT_CLICK, {
+              input_token: token0?.symbol?.toUpperCase(),
+              output_token: token1?.symbol?.toUpperCase(),
+            })
+            window.open(
+              APP_PATHS.KYBERAI_EXPLORE + '/' + SUPPORTED_NETWORK_KYBERAI[chainId] + '/' + token?.address,
+              '_blank',
+            )
+          }}
+        >
+          <RowFit gap="8px">
+            <img src={token?.logo} width="32" height="32" style={{ borderRadius: '50%' }} />
+            <Column gap="4px">
+              <Text color={theme.text}>{token?.symbol?.toUpperCase() || '--'} seems to be</Text>
+              <Text color={color} fontWeight={600}>
+                {token?.kyberScore.label || '--'}
+              </Text>
+            </Column>
+          </RowFit>
+          <SkewKyberScoreWrapper>
+            <KyberScoreMeter
+              value={token.kyberScore.score}
+              noAnimation
+              fontSize="18px"
+              hiddenValue
+              style={{ height: '32px' }}
+            />
+            <RowFit fontSize="10px" gap="4px">
+              <ApeIcon size={14} /> KyberScore
+            </RowFit>
+          </SkewKyberScoreWrapper>
+          <Column gap="4px" padding="0 26px">
+            <Text fontSize="12px">
+              Want to know the <b style={{ color: theme.text }}>KyberScore</b> for{' '}
+              {token?.symbol?.toUpperCase() || 'KNC'}?
+            </Text>
+            <RowFit fontSize="12px" gap="4px">
+              Explore with <span style={{ color: theme.primary }}>KyberAI</span> here!{' '}

```

Comment:
Trans

Reply:
Also, Please use <Text> from 'rebass' to avoid inline style 
`<Text as="span" color={theme.primary }>KyberAI</Text>`